echopop/tests/test_strata_mean_sigma_bs.py

```diff
@@ -0,0 +1,204 @@
+import pandas as pd
+import numpy as np
+from echopop.computation.acoustics import to_linear , to_dB , ts_length_regression
```

Comment:
Suggest renaming `test_strata_mean_sigma_bs.py` to `test_acoustics.py` since all functions included here are directly related to acoustics "fundamentals" and you'll be refactoring `impute_missing_sigma_bs` as well, which is also the same "kind."